EngineeringModel.Tests/app.config

```diff
@@ -1,5 +1,10 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
please undo changes

Reply:
The file has been saved as encoding with signature.